modules/form-field/react/stories/stories_ColorInput.tsx

```diff
@@ -23,6 +23,11 @@ storiesOf('Form Field/Color Picker/Color Input/Top Label', module)
       {controlComponent(<ColorInput disabled={true} />)}
     </FormField>
   ))
+  .add('Required', () => (
+    <FormField label="Label" inputId="input-required" required={true}>
```

Comment:
This isn't actually a variant of the various form elements, so do you mind reverting these extra stories and just adding another `FormField/Label` story? This will require you to rename the current one (i.e. `FormField/Label/Default` and `FormField/Label/Required`)

Reply:
Makes sense! I noticed that there aren't a lot of knobs in the stories, why is that?

Reply:
Yeah - the main reason is that our storybook is integrated into the [Workday design site](http://design.workday.com). It's currently set up to just render different stories, with no access to the knobs. If we put everything behind knobs, we couldn't show the different variants in this context.

We're currently evaluating our entire developer experience (including documentation integrating with storybook), and this is one portion we're looking at. So it's on our radar, but we're not making much use of knobs yet for this reason. Hopefully that answers your question!

Reply:
It does, thanks!